ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -41,65 +31,6 @@ chain:
       echo "Cluster became available, creating kubeconfig"
       bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
 
-      # Enable wildcard routes on the management cluster
-      oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p \
-        '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
-
-      # Waiting for the default NodePort service to be created on the hosted cluster
```

Comment:
Do we need to keep it?

Reply:
The reason to keep this logic would be if you all need to test using clusters <= 4.12.  anything >= 4.13 no longer needs this logic, but <= 4.12 does

Reply:
yes, we do need to keep this. I didn't realize we have 4.12 periodics as well that will neeed to execute this

Reply:
sorry, i have question, if https://github.com/openshift/hypershift/pull/1787 cherry-pike 4.12, we need keep it?